commands/command_env.go

```diff
@@ -44,6 +44,11 @@ func envCommand(cmd *cobra.Command, args []string) {
 	for _, env := range lfs.Environ() {
 		Print(env)
 	}
+
+	for _, key := range []string{"filter.lfs.smudge", "filter.lfs.clean"} {
```

Comment:
Could refer back to [this](https://github.com/github/git-lfs/blob/master/lfs/setup.go#L21) variable